platform/osgi/platform-osgi-condpermadmin/src/test/resources/deny.policy

```diff
@@ -0,0 +1,11 @@
+priority "deny";
```

Comment:
❓ Can we include examples of principal and signer conditions in these test policies?